project_PassMeCash_Polkadot.md

```diff
@@ -0,0 +1,165 @@
+PassMeCash is revolutionizing the banking and financial system by bringing funds to its users upon their request, instead of them going to banks to get funds. Users or Businesses can send and receive payments for goods and services via PassMeCash. Cross border payments and transfer can also be done through PassMeCash. Users who need low fiat currency bills to spray celebrants in parties and social functions, can also request for low currency bills through PassMeCash. All transactions on PassMeCash can be done using crypto (DOT) or cash payments.
+Thus PassMeCash Products are (i) PassMeCash get cash to its users at their choice of location. (ii) PassMeCash help Users and Businesses send and receive payments. (iii) PassMeCash make available low currency bills to her users upon request. (iv) PassMeCash performs Crypto (DOT) to Cash swap and vise versa, for easy transactions.
+The problems addressed by PassMeCash include cash scarcity, low bills requests and crypto to fiat payments, which has affected over 100 million Nigerians in recent time. 
+We intend to build PassMeCash on Polkadot by integrating Python SDK with Ink to execute contract, as regard financial transactions on PassMeCash. Also, crypto (DOT) will be used for transactions which will lead to more adoption of Polkadot and increase number of DOT holders. This will thus add value to the entire ecosystem.
+
+
+### Project Details
+
+Below here are relevant links for PassMeCash towards its designs and developmental stages.
+
+1. Prototype link - https://www.figma.com/proto/tjzAcToD4T8Wm05gGi7Lhu/Client-View?page-id=0%3A1&node-id=220%3A10540&viewport=-28369%2C23670%2C1&scaling=scale-down-width&starting-point-node-id=220%3A10540
+2. Figma link - https://www.figma.com/file/tjzAcToD4T8Wm05gGi7Lhu/Client-View?node-id=0%3A1
+3. Research Data link - https://www.figma.com/file/ES25BCpr4KIazy6xg9w1e4/PassMeCash-Data?node-id=0%3A1
+4. PassMeCash Flow link https://miro.com/app/board/uXjVPqS1sPY=/
+5. PassMeCash Documentation link: https://docs.google.com/document/d/1tgbiOFsb4JE6mHmP2oG-XP5kriGgPhun118o5174e4w/edit?usp=sharing
+6. Github Repo: https://github.com/passmecash
+
+
+The technology stacks to be used include:
+- Figma
+- Miro
+- Tailwind
+- React.js
+- Javascript
+- Python
+- Django
+- Ink
+- Chainlink aggregator
+- Web3.py
+
+
+### Ecosystem Fit
+
+The **PassMeCash** is built for all users of the ecosystem and it can be leveraged upon to send and receive payments globally. Users can also use crypto (DOT) to make transaction payments for goods and services via the PassMeCash platform. This will lead to more adoptability and usability of Polkadot.
```

Comment:
Helping people exchange crypto for local currency is a valiant goal of course, but other than using DOT as a form of payment, it's not clear to me how this app would benefit the overall Polkadot/Kusama ecosystem. Perhaps you could add details about how others can benefit from using your ink! smart contracts? What main functions will be included in the contract(s)?

Reply:
The smart contract automate certain wallets functions, such as sending and receiving payments, which members of the Polkadot/Kusama ecosystem can leverage upon is carrying out their daily transactions. Also, in the long run, we intend to build a PassMeCash API which other developers can then integrate into their applications, and this will aid more projects to be connected to the Polkadot/Kusama ecosystem, as well as the onboarding of more users to the Polkadot/Kusama ecosystem.